### ISSUE TITLE
[Story] Confirm point of contact information for financial institution

### ISSUE THREAD

**Issue by sthomas93** [cfpb/sbl-project#112]
### Epic

Epic issue includes overview, complete list of user stories, technical and content requirements, user testing task ideation, and current design mock-ups.

- https://github.com/cfpb/sbl-project/issues/180

### User story

As a filer,  I want to review the point of contact  information for my filing, so that I can confirm that the information is correct and proceed to finalizing my filing.

### Acceptance criteria

Given a system user has completed all previous steps, when they land on the sign and submit page the system will display a section with point of contact information (First name, last name, phone, email, and address), then the user will be able to review the data for accuracy and completeness.

### Technical tasks

Developers should create technical tasks and add links (below) prior to sprint planning.

#### Front end
- [ ] Task (link)
- [ ] Task (link)

#### Back end
- [ ] Task (link)
- [ ] Task (link)

### Context

![image](https://github.com/cfpb/sbl-project/assets/99903050/73194173-1402-4dd3-a7af-22074e1bad66)

**Comment by natalia-fitzgerald** [comment]
@angelcardoz @sthomas93 
Should a filer be able to provide, confirm, or do both? Will the point of contact information live with the filing data as opposed to within the financial institution profile data? For the confirm is it a review or do they need to check a box or something to indicate that they did a review and are confirming that they did that? 

Do we need a user story for provide and a separate one for confirm? 

Provide: 
"As a filer, I want to be able to provide the point of contact information for my financial institution before I submit my filing, so that the CFPB or other regulator can contact my financial institution about my filing."

Confirm: 
"As a filer, I want to be able to confirm the point of contact information for my financial institution before I submit my filing, so that I can ensure that my financial institution point of contact information is accurate."

+ @dan-padgett

**Comment by dan-padgett** [comment]
@natalia-fitzgerald I think at this point it is "provide" since the sign and submit page is where they would confirm that it is correct (along with all the other information on the sign and submit page).

**Comment by natalia-fitzgerald** [comment]
@sthomas93 @angelcardoz 
What do you think of this change?

**User story:**
As a filer, I want to provide point of contact information for my financial institution prior to signing and submitting my filing, so that the CFPB or other regulators can contact my financial institution about my filing.

**Short version:**
Provide point of contact information for before signing and submitting

**Comment by natalia-fitzgerald** [comment]
@angelcardoz @sthomas93 
In today's backlog grooming follow-up we settled on adding a step to the lending data filing flow called "Provide point of contact". Can we go ahead and create an epic for that? We can then adjust this specific story to capture just the confirm point of contact information that happens at Step 5. Sign and submit. And we can capture the "Provide point of contact" under the other epic.  

**User story:**
As a filer, I want to confirm the contact information for my financial institution prior to signing and submitting, so that the CFPB or other regulators can contact my financial institution about my submission.

**Short version:**
Confirm contact information for before signing and submitting

@dan-padgett

**Comment by sthomas93** [comment]
@natalia-fitzgerald @angelcardoz The epic for Provide point of contact can be found [here](https://github.com/cfpb/sbl-project/issues/150) and the epic for Provide type of financial institution can be found [here](https://github.com/cfpb/sbl-project/issues/143).

**Comment by angelcardoz** [comment]
Changed from "As a filer, I want to be able to confirm POC information for my FI before Sign and Submit, then I will be able to confirm that my information is correct" to "As a filer,  I want to review my point of contact  information, so that I can confirm my information is correct and proceed to finalizing my filing."

**Comment by natalia-fitzgerald** [comment]
Tracking this work in this consolidated story: 
- https://github.com/cfpb/sbl-project/issues/260
